data.sql

```diff
@@ -1,5 +1,65 @@
 /* Populate database with sample data. */
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO vets (name, age, date_of_graduation)
+VALUES
+    ('Vet William Tatcher', 45, '2000-04-23'),
+    ('Vet Maisy Smith', 26, '2019-01-17'),
+    ('Vet Stephanie Mendez', 64, '1981-05-04'),
+    ('Vet Jack Harkness', 38, '2008-06-08');
```

Comment:
- [ ] Dear @hetrox8, You have done great with the project, But when you add the data, I believe you misunderstood how it should be inserted. When we say `Vet William Tatcher is specialized in Pokemon.` it means to show the association between `William Tatcher` and   `Pokemon`, hence when we insert the data, we should only insert `William Tatcher` instead of `Vet William Tatcher`. Please remove all the `Vet ` prefixes for all the names.